batch/batch/server/server.py

```diff
@@ -76,18 +76,60 @@ def make_logger():
 log.info(f'instance_id = {instance_id}')
 
 
+class JobTask:  # pylint: disable=R0903
+    @staticmethod
+    def copy_task(job_id, task_name, files):
+        container = kube.client.V1Container(image='alpine',
+                                            name=task_name,
+                                            command=['echo', 'hello'])
+
+        spec = kube.client.V1PodSpec(containers=[container],
+                                     restart_policy='Never')
+
+        if files:
```

Comment:
Let's move this up so we don't bother creating those two objects if we don't need to.